src/workflow/confirm.c

```diff
@@ -105,11 +105,16 @@ enum workflow_async_ready workflow_confirm_async(
     }
 }
 
-bool workflow_confirm(const char* title, const char* body, bool longtouch, bool accept_only)
+bool workflow_confirm(
```

Comment:
We get more and more combinations of the confirm functions and params. Later we could think about making only one function that takes a config struct instead with good defaults, then calling it is more readable.

https://github.com/digitalbitbox/bitbox02-firmware/pull/253 will also need a new combination where the confirmation buttons are at the top during transaction confirmation :sweat_smile:

Reply:
If you do this, then using the password font here would be possible too:
https://github.com/digitalbitbox/bitbox02-firmware/pull/266#issuecomment-567472999

Reply:
sounds like a good refactoring issue. Can you create it?

Reply:
Done